src/main/resources/swagger/api-docs.yaml

```diff
@@ -1690,6 +1693,58 @@ paths:
         500:
           description: Internal Server Error
 
+  /api/trial/manager/report:
+    post:
+      tags:
+        - Trial
+      operationId: createReportTrialProjects
+      summary: Create a Spreadsheet Report of free trial projects
+      responses:
+        200:
+          description: OK; report data spreadsheet is created
```

Comment:
I am considering a better response for these endpoints. I think it makes sense to provide a link to the created/updated spreadsheet.

Reply:
if you can return the link, 200 OK is fine; if there's no body to the response, maybe 201 Created or 202 Accepted or maybe even 204 No Content